### PR TITLE
Fix Upgrading Guide doc issues

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/upgrading_docs_cleanup

### CODE REVIEW

File: documentation/src/main/asciidoc/topics/upgrading.adoc [hunk 1]
```diff
@@ -413,9 +417,9 @@ The following changes apply to the Infinispan 13 configuration schemas:
 * The `single-file-store` element has been added to the `urn:infinispan:config` namespace if you want to configure a single file store, which is deprecated.
 * The `endpoints` element in the `urn:infinispan:server` namespace is no longer a repeating element, but a wrapper for 0 or more `endpoint` elements.
 
-=== Migrating Soft-Index File Stores
+==== Migrating Soft-Index File Stores
 
-==== Declarative
+===== Declarative
```
Comment (jabolina) [inline documentation/src/main/asciidoc/topics/upgrading.adoc:422]: Because it's nested too deep, it renders it as normal text. Is it OK?

Comment (jabolina) [reply documentation/src/main/asciidoc/topics/upgrading.adoc:422]: Now:

image


Before:

image

Comment (tristantarrant) [reply documentation/src/main/asciidoc/topics/upgrading.adoc:422]: Yeah, I think it's fine

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Thanks, @tristantarrant!

**Pull request merged**